lib/spack/spack/build_systems/cmake.py

```diff
@@ -73,14 +68,12 @@ class CMakePackage(PackageBase):
 
     build_time_test_callbacks = ['check']
 
-    depends_on('cmake', type='build')
-
-    def build_type(self):
-        """Returns the correct value for the ``CMAKE_BUILD_TYPE`` variable
+    # https://cmake.org/cmake/help/latest/variable/CMAKE_BUILD_TYPE.html
+    variant('build_type', default='RelWithDebInfo',
+            description='The build type to build',
```

Comment:
I know this description sucks, but I couldn't think of anything more clear. Any ideas?